src/common/components/signUpModal/signUpModal.component.spec.js

```diff
@@ -909,6 +909,7 @@ describe('signUpForm', function () {
         jest.spyOn($ctrl, 'injectErrorMessages').mockImplementation(() => {});
         jest.spyOn($ctrl, 'injectBackButton').mockImplementation(() => {});
         jest.spyOn($ctrl, 'showVerificationCodeField').mockImplementation(() => {});
+        jest.spyOn($ctrl, 'skipOptionalMFAEnrollment').mockImplementation(() => {});
```

Comment:
Semicolons should be removed.

Reply:
When running `yarn lint:write`, it passes without making any edits or removing the semicolons.

I think we should establish a set rule, as we have some lines with semicolons and some without.

Reply:
The rules are there, but specs are skipped from formatting.

Reply:
@wrandall22 For now, prettier gives me an error when I remove the semicolon, so I will leave it there until we can improve the prettier and lint settings in T3.